core/schema/container.graphqls

```diff
@@ -318,11 +318,16 @@ type Container {
 
   The service dependency will also convey to any files or directories produced by the container.
   """
-  withServiceDependency(
-    service: ContainerID!,
-    "Optional hostname alias to map to the service"
-    alias: String
-  ): Container!
+  withService(service: ContainerID!): Container!
```

Comment:
I really like `withService` and `withServiceAlias`. When I saw it... it felt... _right_. Really nice work on this, @vito.